### PR TITLE
Added toggle and merge operations

### PR DESCRIPTION
**Pull request by Flamenco**
Here's are couple of features we have been using in-house.  Enjoy.

### CODE REVIEW

**Comment by mull** [comment]
@Flamenco did you forget to merge this or are you waiting for review? :)

**Comment by Flamenco** [comment]
I don't have write access to the repo, so I suppose I'm waiting for review ;)

**Comment by mull** [comment]
Oh, oops :)

**Comment by eagleeye** [comment]
@bifrost - you promised to take a look ;)

File: README.md [hunk 1]
```diff
@@ -161,6 +161,39 @@ dotProp.delete(obj, 'foo.0.bar');
 //=> {foo: [{}, 'white-unicorn', 'silver-unicorn']}
 ```
 
+### toggle
+
+Delete a value by a dot path.
```
Comment (bifrost) [inline README.md]: I guess it should be toggle a boolean by a dot path?

**Comment by mistadikay** [comment]
Great work on this one guys! When do you plan to release it?

File: README.md (hunk 1 above)
Comment (Flamenco) [inline README.md]: Yep. Copy and paste got the better of me on that one.

**Comment by eagleeye** [comment]
@mistadikay done!

**Pull request merged**